DESCRIPTION.md

```diff
@@ -10,6 +10,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 - v3.0.3(TBD)
 
+- v3.0.3(April 20,2023)
+
+   -N/A
+
+
   - Fixed a bug that prints error in logs for GET command on GCS.
   - Added a parameter that allows users to skip file uploads to stage if file exists on stage and contents of the file match.
   - Fixed a bug when writing a Pandas DataFrame with non-default index in `snowflake.connector.pandas_tool.write_pandas`.
```

Comment:
Fixed a bug that occurred when ...

Reply:
@sfc-gh-mheavin  Please commit the recommendation here in this branch.

Reply:
@sfc-gh-kdama I don't have the git repo yet. Whoever added the notes can choose to include my suggestion.

Reply:
```suggestion
  - Fixed a bug that occurred when writing a Pandas DataFrame with non-default index in `snowflake.connector.pandas_tool.write_pandas`.
```

Reply:
@sfc-gh-mkeller Can I just commit here directly in git instead of cloning the repo and checking out the branch?

Reply:
you can do that to @sfc-gh-mheavin  going forward.